src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -278,6 +278,18 @@
       <arg type="s" name="transaction_address" direction="out"/>
     </method>
 
+   <!-- Available options:
+        "editor" (type 'b')
+        "merge" (type 'b')
+    -->
+    <method name="KernelArgs">
```

Comment:
This feels close to the `SetInitramfsState` method - I don't see a reason for it to be experimental either.  So let's move this right underneath that method in the XML, and also implement it close by in `rpmostreed-os.c`?

We might actually consider a general `SetBootConfiguration` method, but we don't need to do that right now.  Maybe though internally route `os_handle_set_initramfs_state()` and `os_handle_set_boot_args()` into an internal-only `os_handle_boot_configuration()` method or so at least?  Like how we route the pkg install/deploy etc. into a generic `os_merge_or_start_deployment_txn()`.

(And really perhaps down the line we need to be able to both do initramfs/kargs/deploy all in one, but no need to worry about that now)